application/classes/Ushahidi/Repository/Form/Attribute.php

```diff
@@ -31,6 +34,12 @@ public function create(Entity $entity)
 	{
 		$record = $entity->asArray();
 		unset($record['form_id']);
+		try {
+			$uuid = Uuid::uuid4();
+			$record['key'] = $uuid->toString();
```

Comment:
Could we keep `key` if its passed manually? or does that leave us with collisions again?